src/testsets.jl

```diff
@@ -11,6 +11,7 @@ mutable struct RAITestSet <: Test.AbstractTestSet
     report::Bool
     distributed::Bool
     distributed_tests::Vector{Task}
+    serial_test_results::Vector{Any}
```

Comment:
I don't think I'm following why this is needed

Why don't the normal record functions catch what we need here?